ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -1167,6 +1171,76 @@ def test_ds_configcheck_passwordstorage(self, modify_pwdstoragescheme):
                 assert "cn=config" in check["kw"]["items"]
                 assert error_msg in check["kw"]["msg"]
 
+    @pytest.fixture
+    def create_logfile(self):
+        """
+        This fixture creates an empty healthcheck.log file when
+        it is not present in /var/log/ipa/healthcheck/ directory so that
+        sosreport command collects the log file in the report generated
+        by the dry run and then deletes the file once the test is finished.
+        """
+        if not self.master.transport.file_exists(HEALTHCHECK_LOG):
+            self.master.run_command(
+                ["ipa-healthcheck", "--output-file", HEALTHCHECK_LOG]
+            )
+        yield
+        self.master.run_command(["rm", HEALTHCHECK_LOG])
```

Comment:
Add "-f" option to make sure the file is removed without prompting for confirmation.